frontend/packages/client/package.json

```diff
@@ -32,8 +32,8 @@
     "react-query": "^3.39.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
-    "string-strip-html": "8.3.0",
-    "sass": "^1.49.8"
+    "sass": "^1.49.8",
+    "string-strip-html": "8.3.0"
   },
   "scripts": {
     "start": "react-scripts start",
```

Comment:
can you remove `package-lock.json` and run `yarn install` ?

Reply:
sure thing